test-common/uk/gov/hmrc/alcoholdutyaccount/common/TestData.scala

```diff
@@ -270,6 +600,8 @@ trait TestData extends ModelGenerators {
   val adrMultipleOpenData =
     Seq(adrObligationDetails, adrObligationDetailsOpen2)
 
+  val openPayments = OpenPayments(Seq.empty, BigDecimal(0), Seq.empty, BigDecimal(0), BigDecimal(0))
```

Comment:
Is this still in use? If so better name and field assignments - however I think it was a stop-gap.